ios/FluentUI/Command Bar/CommandBarTokens.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class MSFCommandBarTokens: MSFTokensBase {
+    public var backgroundColor: UIColor!
+    public var iconColor: UIColor!
+
+    public var hoverBackgroundColor: UIColor!
```

Comment:
where is hover color being used? keyboard focus? pointer color?

Reply:
Updated with pointer usage